Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /root
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
-    apt-get install -y curl groovy nodejs openjdk-8-jre-headless php python ruby
+    apt-get install -y curl groovy nodejs openjdk-8-jre-headless php python ruby gawk
```

Comment:
oops didn't realize the alphabetic order here

Reply:
No worries, I moved it in the merge commit.